web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp

```diff
@@ -15,318 +15,317 @@
 
 <!-- Preview CRF Version for <c:out value="${section.crf.name}" /> <c:out value="${section.crfVersion.name}" /> -->
 <c:if test="${studySubject != null && studySubject.id>0}">
-  <c:choose>
-    <c:when test="${EventCRFBean.stage.name=='initial data entry'}">
-	   <img src="images/icon_InitialDE.gif" alt="Initial Data Entry" title="Initial Data Entry">
-    </c:when>
-    <c:when test="${EventCRFBean.stage.name=='initial data entry complete'}">
-	   <img src="images/icon_InitialDEcomplete.gif" alt="Initial Data Entry Complete" title="Initial Data Entry Complete">
-    </c:when>
-    <c:when test="${EventCRFBean.stage.name=='double data entry'}">
-	   <img src="images/icon_DDE.gif" alt="Double Data Entry" title="Double Data Entry">
-    </c:when>
-    <c:when test="${EventCRFBean.stage.name=='data entry complete'}">
-	   <img src="images/icon_DEcomplete.gif" alt="Data Entry Complete" title="Data Entry Complete">
-    </c:when>
-	<c:when test="${EventCRFBean.stage.name=='administrative editing'}">
-	   <img src="images/icon_AdminEdit.gif" alt="Administrative Editing" title="Administrative Editing">
-    </c:when>
-    <c:when test="${EventCRFBean.stage.name=='locked'}">
-	   <img src="images/icon_Locked.gif" alt="Locked" title="Locked">
-    </c:when>
-    <c:otherwise>
-	   <img src="images/icon_Invalid.gif" alt="Invalid" title="Invalid">
-	</c:otherwise>
-  </c:choose>
-  </c:if>
+    <c:choose>
+        <c:when test="${EventCRFBean.stage.name=='initial data entry'}">
+            <img src="images/icon_InitialDE.gif" alt="<fmt:message key="initial_data_entry" bundle="${resword}"/>" title="<fmt:message key="initial_data_entry" bundle="${resword}"/>">
+        </c:when>
+        <c:when test="${EventCRFBean.stage.name=='initial data entry complete'}">
+            <img src="images/icon_InitialDEcomplete.gif" alt="<fmt:message key="initial_data_entry_complete" bundle="${resword}"/>" title="<fmt:message key="initial_data_entry_complete" bundle="${resword}"/>">
+        </c:when>

```

Comment:
This key translates to: "Initial Data Entry Completed".  Need to create a new one that translates to "Initial Data Entry Complete"